xtext-website/documentation/103_domainmodelnextsteps.md

```diff
@@ -235,46 +285,59 @@ This tutorial is about testing the parser and the linker for the *Domainmodel*.
             assertSame(entity, entity.features.head.type)
         }
     ```
+2. In addition, the utility class [ValidationTestHelper]({{site.src.xtext_core}}/org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/validation/ValidationTestHelper.java) allows to test the custom validation rules written for the language. Both valid and invalid models can be tested.
+
+    ```xtend      
```

Comment:
Please remove the unnecessary trailing white spaces after the `xtend` word.

Reply:
Please use consistent indentation (either only whitespaces or tabs) in the code snippets, do not mix them!

Reply:
Please also add `@Inject ParseHelper<Domainmodel> parseHelper` to make the example more complete.